piscord/Gateway.py

```diff
@@ -2,110 +2,118 @@
 import json
 import asyncio
 
-from .Errors import TokenError,ConnexionError
+from .Errors import TokenError, ConnexionError
+
 
 class Gateway:
 
-	def __init__(self, url, token, auth_op = 10, events_code = 0, presence = None):
-		self.url = url
-		self.token = token
-		self.auth_op = auth_op
-		self.last_sequence = 0
-		self.events_code = events_code
-		self.session_id = None
-		self.presence = presence
-		self.error = None
+    def __init__(self, url, token, auth_op=10, events_code=0, heartbeat_code=1, presence=None):
+        self.url = url
+        self.token = token
+        self.auth_op = auth_op
+        self.last_sequence = 0
+        self.events_code = events_code
+        self.heartbeat_code = heartbeat_code
+        self.presence = presence
+        self.session_id = None
+        self.error = None
+        self.heartbeat = None
```

Comment:
Add a few None initialization and move all the None initialization at the bottom of the __init__ method